[CMSP-459] Updates behat tests and behaviors.

rwagner00 opened the pull request:
Ports behat fixes from https://github.com/pantheon-systems/wp-redis/pull/429/files into its own PR.

This PR also removes the `terminus redis:enable` step in `bin/behat-prepare.sh`. There's no reason to ever _disable_ redis on the fixture running Behat tests, and the enable step spawns many duplicate "redis enable" requests, most of which time-out, causing the tests to fail.

Tests can be seen as passing (with #437) here: https://app.circleci.com/pipelines/github/pantheon-systems/wp-redis/1629/workflows/fd714abf-075f-45f8-9102-3ddc730f731e/jobs/4980

Commit:
Updates behat tests and behaviors.

Commit:
Updates behat tests and behaviors.

pwtyler commented:
Are some of these tests specific to #426 and therefore now failing on this branch?

jazzsequence committed:
port diff from #426

jazzsequence committed:
tabs to spaces

jazzsequence committed:
don't enable redis as part of the prepare step
we end up making a dozen differnet enable redis requests and inevitably a lot of those fail because there are so many happening simultaneously (assumption). It's okay to just enable it and leave it on, rather than turning it on and off.

pwtyler reviewed (approved):
LGTM, as I understand it takes merging both this and #437 to 🟢 behat on default branch
